src/arrayMethodReduce.js

```diff
@@ -6,6 +6,19 @@
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
     // write code here
+    let prev = initialValue;
```

Comment:
```suggestion
    let accumulator = initialValue;
```

Reply:
It's not just a `prev` - it's an `accumulator` for accumulating reduce callback results